### PR TITLE
ISPN-16247: Add more info about pessimistic transactions locking mode

### PR DESCRIPTION
**Pull request by theashiot**
https://issues.redhat.com/browse/ISPN-16247

### CODE REVIEW

**Comment by theashiot** [comment]
Hi @pruivo,

Can you please review the update?

thanks,
ashwin

File: documentation/src/main/asciidoc/topics/ref_hotrod_client_transactions.adoc [hunk 1]
```diff
@@ -18,6 +18,12 @@ The following server configuration is required, otherwise transactions rollback
 
 * Isolation level must be `REPEATABLE_READ`.
 * `PESSIMISTIC` locking mode is recommended but `OPTIMISTIC` can be used.
++
+[NOTE]
+====
+Embedded {brandname} supports pessimistic locks but HotRod clients do not. Therefore, the transaction result obtained from using pessimistic locks in {brandname} server might differ from the result obtained from HotRod client.
```
Comment (pruivo) [inline documentation/src/main/asciidoc/topics/ref_hotrod_client_transactions.adoc:24]: I would move this to line 44, inside section `== Configuring Hot Rod Clients`.

typo, hot rod is 2 words :)  `HotRod` -> `Hot Rod`

Comment (theashiot) [reply documentation/src/main/asciidoc/topics/ref_hotrod_client_transactions.adoc]: Thanks, @pruivo, updated! WDYT?

**Comment by pruivo** [comment]
merged! thanks @theashiot !

**Pull request merged**